scripts/scanDependents/README.md

```diff
@@ -32,32 +32,26 @@ Pass it to the command like this
 yarn scan-dependents --token YOUR_PERSONAL_ACCESS_TOKEN
 ```
 
-## Scan versions of the design systems in use
+## Common formulas
 
-This is the default behavior of the script if no additional options are specified, the script will print out design system versions in use by our dependents.
-
-## Scan dependencies of dependents
-
-You can scan for which versions of certain dependencies our dependents are using with
+### Scan for the version of React our dependencies are using
 
 ```
-yarn scan-dependents --dependency <dependency-package-name>
+yarn scan-dependents --dependency react
 ```
 
-For example, this would scan for which version of React our dependencies are using:
+### Scan only a subset of our design systems
 
 ```
-yarn scan-dependents --dependency react
+yarn scan-dependents --designSystems @cmsgov/ds-medicare-gov @cmsgov/ds-healthcare-gov --dependency node-sass
```

Comment:
These docs don't work for me now that `--dependency` has changed to just `dependency..`

this works:

`yarn scan-dependents dependency node-sass --designSystems @cmsgov/ds-medicare-gov @cmsgov/ds-healthcare-gov`